doc/nrf/ug_multi_image.rst

```diff
@@ -81,6 +77,8 @@ You can enable existing child images in the |NCS| by enabling the respective mod
 To turn an application that you have implemented into a child image that can be included in a parent image, you must update the build scripts to make it possible to enable the child image and add the required configuration options.
 You should also know how image-specific variables are disambiguated and what targets of the child images are available.
 
+If a child image is to be executed on a different core, the *domain* corresponding to that core must be specified when adding the child image.
```

Comment:
The term used in the build system and partition manager for a separate name-space with a different address space is a "domain". This is specified in the partition manager documentation. When you use the `add_child_image` CMake function to add a child image, you can add a `DOMAIN` argument, as seen here:

```
    add_child_image(
      NAME hci_rpmsg
      SOURCE_DIR ${ZEPHYR_BASE}/samples/bluetooth/hci_rpmsg
      DOMAIN nrf5340pdk_nrf5340_cpunet)
```

Reply:
I assume the users will see the example and see that they can use the core/board name as the domain. Currently the name of the domain must match the board name which the images in that domain is build for. This will change in the future however, allowing non-board domains to be specified.

You could use domains in a non-multi-core build if you want to have two dynamic regions (like 'app') within the subset of a single core.

Reply:
Well, let's trust that the user will figure that out then. I wouldn't have any idea how to do it.

Reply:
> Still content missing, but we won't be able to fix that within a day.

If you have concerns in addition to those mentioned here already please report them at NCSDK-5702